tests/ts/blkid/dm-err

```diff
@@ -33,6 +33,15 @@ DEVICE="/dev/mapper/${DEVNAME}"
 ts_log "Initialize device"
 echo "0 262144 error" | dmsetup create $DEVNAME
 
+# Some tests are run in LXD containers on Travis CI where `mknod` doesn't always work
+# https://docs.travis-ci.com/user/multi-cpu-architectures/#security-and-lxd-container
+# https://linuxcontainers.org/lxd/docs/master/syscall-interception
+if [ "$TRAVIS" == "true" ]; then
+	if ! [ -b "$DEVICE" ]; then
```

Comment:
I'm just trying to figure out whether it works. I'll replace it with something like `[ ... ] && [ ... ]` later.